src/neuroglancer/datasource/render/frontend.ts

```diff
@@ -390,8 +392,133 @@ export function volumeCompleter(
+    let projectInfo = ownerInfo.projects.get(project);
+    if (projectInfo === undefined) {
+      throw new Error(
+          `Specificed project ${JSON.stringify(project)} does not exist for specified owner ${JSON.stringify(ownerInfo.owner)}`);
+    }
+
+    if (stack === undefined) {
+      const stackNames = Array.from(projectInfo.stacks.keys());
+      if (stackNames.length !== 1) {
+        throw new Error(`Dataset contains multiple stacks: ${JSON.stringify(stackNames)}`);
+      }
+      stack = stackNames[0];
+    }
+    const stackInfo = projectInfo.stacks.get(stack);
+    if (stackInfo === undefined) {
+      throw new Error(
+          `Specified stack ${JSON.stringify(stack)} is not one of the supported stacks: ` +
+          JSON.stringify(Array.from(projectInfo.stacks.keys())));
+    }
+    this.stack = stack;
+    this.stackInfo = stackInfo;
+
+    let matchCollection = verifyOptionalString(parameters['matchCollection']);
+    if (matchCollection === undefined) {
+      matchCollection = stack;
+    }
+    this.matchCollection = matchCollection;
+
+    let zoffset = verifyOptionalInt(parameters['zoffset']);
+    if (zoffset === undefined) {
+      zoffset = 1;
+    }
+    this.zoffset = zoffset;
+
+    this.dims = vec3.create();
+
+    let tileSize = verifyOptionalInt(parameters['tilesize']);
+    if (tileSize === undefined) {
+      tileSize = 1024;  // Default tile size is 1024 x 1024
+    }
+    this.dims[0] = tileSize;
+    this.dims[1] = tileSize;
+    this.dims[2] = 1;
+  }
+  getSources(vectorGraphicsSourceOptions: VectorGraphicsSourceOptions) {
+    let sources: VectorGraphicsChunkSource[][] = [];
+
+    let voxelSize = vec3.clone(this.stackInfo.voxelResolution);
+
+    let lowerVoxelBound = vec3.create(), upperVoxelBound = vec3.create();
+
+    for (let i = 0; i < 3; i++) {
+      lowerVoxelBound[i] = Math.floor(
+          this.stackInfo.lowerVoxelBound[i] * (this.stackInfo.voxelResolution[i] / voxelSize[i]));
+      upperVoxelBound[i] = Math.ceil(
+          this.stackInfo.upperVoxelBound[i] * (this.stackInfo.voxelResolution[i] / voxelSize[i]));
+    }
+
+    // For now we set the chunkDataSize to be the size of an entire slab, pending possible bug fix
+    // in render point match service
+    let chunkDataSize = vec3.clone(upperVoxelBound);
+    chunkDataSize[0] += Math.abs(lowerVoxelBound[0]);
+    chunkDataSize[1] += Math.abs(lowerVoxelBound[1]);
+    chunkDataSize[2] = 1;
+
+
+    let spec = VectorGraphicsChunkSpecification.make(
+        {voxelSize, lowerVoxelBound, upperVoxelBound, chunkDataSize, vectorGraphicsSourceOptions});
+    let source = PointMatchSource.get(this.chunkManager, spec, {
+      'baseUrls': this.baseUrls,
+      'owner': this.ownerInfo.owner,
+      'project': this.stackInfo.project,
+      'stack': this.stack,
+      'encoding': 'points',
+      'matchCollection': this.matchCollection,
+      'zoffset': this.zoffset
+    });
+
+    return [[source]];
+  }
+};
```

Comment:
semicolon not needed (I know other neuroglancer code also has redundant semicolons)